modules/spring-config/src/main/java/org/mule/runtime/config/internal/dsl/spring/ObjectFactoryClassRepository.java

```diff
@@ -127,4 +180,33 @@
     });
     return factoryBeanClass;
   }
+
+  public class IsEagerInitGetterInterceptor implements InvocationHandler {
```

Comment:
make this private

Reply:
It cannot be private because the generated class needs to access it. I made it protected.